### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [aidenlx/obsidian-zotero#158] [Announcements]
# index

https://obzt.aidenlx.top/

**Comment by bepolymathe** [comment]
Great project. Thanks !

**Reply by flashshare** [reply]
Thanks for the plugin. Very usel

**Comment by lancewilhelm** [comment]
Thanks for the plugin! Looking forward to more features, but it already accomplishes a lot of what I need. Feels a bit better than the other zotero integration plugin.

**Comment by Lukkarius** [comment]
First, thanks for the wonderful plugin. Love it!!

Question, how can i handle Note with i have in zotero.
I know you have a action to create a seperated file with the note. But is it possible in the same item folder like this

metadata
# heading
## Notes
### Note 1
Note....
### Note 2
...
## Highlights
...


Thanks

**Comment by pocketdollhouse** [comment]
I can't install the zotero plugin on firefox, even after downloading it by clicking 'save as'. It says that the add-on couldn't be installed because it is corrupt.

**Reply by pocketdollhouse** [reply]
(Sorry - I realised I was supposed to install it on Zotero, not Firefox! Here are some instructions for others in case you are similarly confused)

Install by downloading the latest release – if you use Firefox, make sure to right-click and save the XPI file, not just clicking it – and then in Zotero:

    In the main menu go to Tools > Plugins
    Select ‘Plugins’
    Click on the gear in the top-right corner and choose ‘Install Plugin From File…’
    Choose .xpi that you’ve just downloaded, click ‘Install’
